src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -11466,12 +11434,6 @@ void Dummy()
             var compilation = CreateCompilation(new[] { source }, options: WithNonNullTypesTrue());
 
             compilation.VerifyDiagnostics(
-                // (19,50): warning CS8608: Nullability of reference types in type doesn't match overridden member.
```

Comment:
>// (19,50): warning CS8608: Nullability of reference types in type doesn't match overridden member. [](start = 16, length = 99)

I would expect both warnings to remain.

Reply:
Does that mean that events are invariant through overriding? That makes sense to me.

Reply:
> Does that mean that events are invariant through overriding?

I do not think so. Why?

---
In reply to: [273354165](https://github.com/dotnet/roslyn/pull/34718#discussion_r273354165) [](ancestors = 273354165)